dashboard-ui/src/components/Survey/dynamic.jsx

```diff
@@ -134,7 +134,10 @@ const Dynamic = ({ patients, situation, supplies, decision, dmName, actions, exp
                                     <Accordion.Body>
                                         <ListGroup>
                                             {actions.map((action, index) => (
-                                                <ListGroup.Item key={"action-" + index}>{action}</ListGroup.Item>
+                                                <ListGroup.Item key={"action-" + index} style={{
+                                                    "font-weight": action.includes('Update:') || action.includes('Note:') ? "700" : "500",
```

Comment:
This bit here was causing console errors for me when taking the survey. 
<img width="333" alt="Screenshot 2024-07-10 at 2 54 01 PM" src="https://github.com/NextCenturyCorporation/itm-evaluation-dashboard/assets/73036954/c7b6e232-52e9-44e5-9595-b6824346eba4">
<img width="363" alt="Screenshot 2024-07-10 at 2 53 30 PM" src="https://github.com/NextCenturyCorporation/itm-evaluation-dashboard/assets/73036954/576f39a2-1f6e-49c5-898e-0f2986a5dd0f">

Changing them to `fontWeight` and `backgroundColor` resolved for me locally.

Reply:
Did you want to commit that change?

Reply:
Just pushed it